Usee $(CC) to enable cross compiling and link to the right ncurses library.

Signed-off-by: Qais Yousef <qais.yousef@imgtec.com>

--- latencytop-0.5/Makefile.old	2012-04-18 11:59:39.748084334 +0100
+++ latencytop-0.5/Makefile	2012-04-18 12:00:08.659112340 +0100
@@ -4,7 +4,7 @@
 DESTDIR =
 SBINDIR = /usr/sbin
 XCFLAGS = -W  -g `pkg-config --cflags glib-2.0` -D_FORTIFY_SOURCE=2 -Wno-sign-compare
-LDF = -Wl,--as-needed `pkg-config --libs glib-2.0`   -lncursesw 
+LDF = -Wl,--as-needed `pkg-config --libs glib-2.0`   -lncurses 
 
 OBJS= latencytop.o text_display.o translate.o fsync.o
 
@@ -26,10 +26,10 @@
 
 # We write explicity this "implicit rule"
 %.o : %.c
-	gcc -c $(CFLAGS) $(XCFLAGS) $< -o $@
+	$(CC) -c $(CFLAGS) $(XCFLAGS) $< -o $@
 
 latencytop:  $(OBJS) latencytop.h Makefile
-	gcc $(CFLAGS) $(OBJS) $(LDF) -o latencytop 
+	$(CC) $(CFLAGS) $(OBJS) $(LDF) -o latencytop 
 
 clean:
 	rm -f *~ latencytop DEADJOE *.o
